wagtail/contrib/wagtailfrontendcache/tests.py

```diff
@@ -43,6 +46,41 @@ def test_cloudflare(self):
         self.assertEqual(backends['cloudflare'].cloudflare_email, 'test@test.com')
         self.assertEqual(backends['cloudflare'].cloudflare_token, 'this is the token')
 
+    def test_cloudfront(self):
+        backends = get_backends(backend_settings={
+            'cloudfront': {
+                'BACKEND': 'wagtail.contrib.wagtailfrontendcache.backends.CloudfrontBackend',
+                'DISTRIBUTION_ID': 'frontend',
+            },
+        })
+
+        self.assertEqual(set(backends.keys()), set(['cloudfront']))
+        self.assertIsInstance(backends['cloudfront'], CloudfrontBackend)
+
+        self.assertEqual(backends['cloudfront'].cloudfront_distribution_id, 'frontend')
+
+    def test_cloudfront_validate_distribution_id(self):
+        with self.assertRaises(ImproperlyConfigured):
+            get_backends(backend_settings={
+                'cloudfront': {
+                    'BACKEND': 'wagtail.contrib.wagtailfrontendcache.backends.CloudfrontBackend',
+                },
+            })
+
+    @mock.patch('wagtail.contrib.wagtailfrontendcache.backends.CloudfrontBackend._create_invalidation')
+    def test_cloudfront_distribution_id_mapping(self, _create_invalidation):
+        backends = get_backends(backend_settings={
+            'cloudfront': {
+                'BACKEND': 'wagtail.contrib.wagtailfrontendcache.backends.CloudfrontBackend',
+                'DISTRIBUTION_ID': {
+                    'www.wagtail.io': 'frontend',
+                }
+            },
+        })
+        backends.get('cloudfront').purge('http://www.wagtail.io/home/events/christmas/')
+
+        _create_invalidation.assert_any_call('frontend', '/home/events/christmas/')
+
```

Comment:
Could we have a test case for when a cache invalidation is performed on a domain that isn't in the ``DISTRIBUTION_ID`` mapping?